odl/solvers/advanced/proximal_operators.py

```diff
@@ -143,26 +251,28 @@ def make_prox(tau):
 
         return IdentityOperator(space)
 
-    return make_prox
+    return make_identity
 
 
-def proximal_nonnegativity(space):
-    """Function to create the proximal operator of G(x) = ind(x > 0).
+def proximal_clamp(space, lower=None, upper=None):
```

Comment:
I'm not so happy with the name "clamp". Is it common? I never read it before. Most people will be more likely to recognize something like `proximal_box_constraint`.